planb/remote_command.py

```diff
@@ -0,0 +1,67 @@
+import click
+import subprocess
+
+from .common import boto_client, list_instances
+
+
+def quoted(command: str) -> str:
+    return '"{}"'.format(
+        command.replace("\\", "\\\\").replace("\"", "\\\"").replace("$", "\\$")
+    )
+
+
+def run_shell(
+        command: list, cluster_name: str, region: str, odd_host: str, piu: str,
+        echo: bool, no_prompt: bool, no_wait: bool, ip_label: bool):
+
+    ec2 = boto_client('ec2', region)
+    instances = list_instances(ec2, cluster_name)
+    if not instances:
+        click.echo(
+            "No running instances found in region {} with Name tag '{}'".format(
+                region,
+                cluster_name
+            ),
+            err=True
+        )
+    for i in instances:
```

Comment:
Could we make it fit on one page?

Reply:
It *does* fit on my page:

![screenshot from 2018-02-20 09-55-29](https://user-images.githubusercontent.com/489601/36414907-41700d7c-1624-11e8-9f3f-1121bc9e6491.png)

Reply:
Still not very readable. I'd propose to put the statements in the main loop into a function.  This would also reduce the level of nesting by one.